Sources/ApolloCodegenLib/Glob.swift

```diff
@@ -191,7 +191,7 @@ public struct Glob {
       globfree(&globT)
     }
 
-    CodegenLogger.log("Matching \(pattern)", logLevel: .debug)
+    CodegenLogger.log("Evaluating \(pattern)", logLevel: .debug)
```

Comment:
`Matching` was too close to `Matched`. `Evaluating` is more indicative of what the logic is actually doing.